screens/Navigation/ConversationRequestsListNav.ios.tsx

```diff
@@ -84,26 +84,25 @@ export default function ConversationRequestsListNav() {
 
   const navigationOptions = useCallback(
     ({
+      route,
       navigation,
     }: {
       route: RouteProp<NavigationParamList, "ChatsRequests">;
-      navigation: any;
+      navigation: NativeStackNavigationProp<
+        NavigationParamList,
+        "ChatsRequests"
+      >;
     }) => ({
       animation: navigationAnimation as StackAnimationTypes,
       headerTitle: clearingAll
-        ? Platform.OS === "ios"
-          ? () => (
-              <View style={styles.headerContainer}>
-                <ActivityIndicator />
-                <Text style={styles.headerText}>Clearing</Text>
-              </View>
-            )
-          : "Clearing..."
+        ? () => (
+            <View style={styles.headerContainer}>
+              <ActivityIndicator />
+              <Text style={styles.headerText}>Clearing</Text>
```

Comment:
Mind putting some of these strings into translations?
https://github.com/ephemeraHQ/converse-app/blob/main/i18n/translations/en.ts

Would change to translate("clearing")
imported from '@i18n'